src/Worker/Core/Shims/TaskActivityShim.cs

```diff
@@ -3,7 +3,7 @@
 
 using DurableTask.Core;
 
-namespace Microsoft.DurableTask;
+namespace Microsoft.DurableTask.Worker.Shims;
```

Comment:
Looking at the classes in this file, are they redundant with the `FuncTaskActivity` code you're adding?